py3status/modules/arch_updates.py

```diff
@@ -63,7 +67,8 @@ def post_config_hook(self):
             self.include_pacman = True
 
         # check cower installed
-        if self.include_aur and not self.py3.check_commands(['cower']):
+        if self.include_aur and self.check_aur_cmd == 'cower -u' and \
+                not self.py3.check_commands(['cower']):
```

Comment:
If we do `command = self.check_aur_cmd.split()[0]`, then we could do this too...
`not self.py3.check_commands(command):` and `self.py3.notify_user('%s is not installed
cannot check aur' % command)`.